### ISSUE TITLE
Add training resources / tutorials

### ISSUE THREAD

**Issue by shiffman** [ml5js/training-lstm#7]
Paperspace:
https://blog.paperspace.com/training-an-lstm-and-using-the-model-in-ml5-js/

Spell:
https://youtu.be/xfuVcfwtEyw

**Comment by jchwenger** [comment]
Here is a [Colab Notebook](https://drive.google.com/file/d/1WkTEDaaYzP5z3S3lQB2ADoM1cYq7uQlJ/view?usp=sharing) with instructions on how to train it easily directly inside your Google Drive.

**Comment by wincentbalin** [comment]
> 
> 
> Here is a [Colab Notebook](https://drive.google.com/file/d/1WkTEDaaYzP5z3S3lQB2ADoM1cYq7uQlJ/view?usp=sharing) with instructions on how to train it easily directly inside your Google Drive.

This Colab Notebook points to the non-existent repository https://github.com/ml5/training-lstm.

**Comment by jchwenger** [comment]
@wincentbalin you're right, I updated it to reflect the current state of affairs, it should work now.

**Comment by shiffman** [comment]
Years later @jchwenger I am just taking a look at this now. I've saved it into my NYU drive for possible use with students and will be testing / making some changes. Thank you for sharing!!!

https://colab.research.google.com/drive/1V1xJfHfoG0UrI4Og3sE4kG2De1gLg0NK?authuser=3

**Comment by jchwenger** [comment]
Much better late than never @shiffman, thanks so much for your answer! I learnt so much from you, it's a joy to contribute even in such a modest fashion. I hope everything works fine, let me know if there's anything else I can do to help.
